pages/folder.tsx

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from "react";
+import Banner from "../components/domains/folder/Banner";
+import CardList from "../components/commons/CardList";
+import SearchInput from "../components/commons/SearchInput";
+import styles from "../styles/folderPage.module.css";
+import FolderButtonList from "../components/domains/folder/FolderButtonList";
+import FolderTitles from "../components/domains/folder/folderTitle/FolderTitles";
+import FloatingButton from "../components/domains/folder/FloatingButton";
+import { getAllLinksData, getFoldersData } from "./api/FolderApi";
+import { Folder } from "../types/folder";
+import { Link } from "../types/common";
+
+function FolderPage({ openModal }: { openModal: () => void }) {
+  const [folderList, setFolderList] = useState<Folder[]>([]);
+  const [selectFolderLinks, setSelectFolderLinks] = useState<Link[]>([]);
+  const [id, setId] = useState<number>(0);
+  const [searchLinks, setSearchLinks] = useState<Link[]>([]);
+  const [searchResult, setSearchResult] = useState<string>("");
+
+  const handleFoldersLoad = async () => {
+    const allLinksFolder: Folder = {
+      id: 0,
+      favorite: false,
+      name: "전체",
+      user_id: 1,
+      links: [
+        {
+          id: 0,
+          url: "",
+          title: "",
+          image_source: "",
+          created_at: "",
+        },
+      ],
+    };
+    const allData = await getAllLinksData();
+    const { data } = await getFoldersData();
+    allLinksFolder.links = allData.data;
+    setFolderList([allLinksFolder, ...data]);
+  };
+
+  const searchLink = async (keyword: string) => {
+    const filteredLinks = folderList[0].links?.filter(
+      (link: Link) =>
+        link.url?.includes(keyword) ||
+        link.title?.includes(keyword) ||
+        link.description?.includes(keyword)
+    );
+    setSearchLinks(filteredLinks);
+  };
+
+  useEffect(() => {
+    handleFoldersLoad();
+  }, [id]);
+
+  return (
+    <>
+      <Banner openModal={openModal} />
```

Comment:
지금 표현보다, 기능과 정체성이 잘 드러나면 좋겠습니다.
지금은 그저 배너UI가 있는 데, 알 수는 없지만 _어쨋든 뭔가 아무튼 어떤_ 모달을 여는 기능이 어딘가 달려있는 거같은 느낌을 줍니다.

그리고 내용면에서는 text input과 button을 가진 일종의 단순한 form이라서 이렇게 네이밍하고, 안에서 모달띄워서 text input내용 받아서 api콜하는 기능도 그냥 전부 안에서 책임이지고 할 수 있을거같아요. 그럼 이름값하는 컴포넌트가 될거같습니다
```suggestion
      <AddLinkBanner /> // or AddLinkForm
```